build.gradle.kts

```diff
@@ -1,20 +1,13 @@
-import com.adarshr.gradle.testlogger.theme.ThemeType
-import org.jetbrains.intellij.tasks.RunPluginVerifierTask.VerificationReportsFormats.*
-import org.jetbrains.intellij.tasks.RunPluginVerifierTask.FailureLevel.*
-
 fun properties(key: String) = providers.gradleProperty(key)
 fun environment(key: String) = providers.environmentVariable(key)
 fun prop(name: String): String {
     return properties(name).get()
 }
+
```

Comment:
Same here. This is mostly the Gradle 2.x build script with exactly what was needed from the Gradle 1.x build script for the build/test/run tasks to run.